tests/test-sources/plugins/by-name/efmls-configs/default.nix

```diff
@@ -21,7 +21,10 @@
         "JSON"
       ];
 
-      brokenTools = [ ];
+      brokenTools = [
+        # deprecated
+        "typstfmt"
+      ];
```

Comment:
It's not just deprecated, the package is completely removed (it's now a `throw`).

Doesn't efmls have a packages list used for auto install? We should drop typstfmt there instead of just skipping it in our tests.

Reply:
Now it's in the file twice; under `unpackaged` and top-level `packages`

Reply:
Wow.
I'm slow today. Done.